src/PerformanceMonitor/README.md

```diff
@@ -0,0 +1,30 @@
+It is a plugin to create performance metrics used to benchmark MEO3. You can follow the instructions on [neo-modules](https://github.com/neo-project/neo-modules/blob/master/README.md) to install the plugin.
```

Comment:
I think you meant "Neo3" instead of "Meo3"

Reply:
Yes, sorry for the typo.